test/services/storage/storage.spec.ts

```diff
@@ -0,0 +1,202 @@
+import chai from 'chai'
```

Comment:
I would rename the file to `processor.spec.ts` as you are unit-testing the processor component here and it is already nested in `storage` folder ;-)